## Accessing genomic reference and experimental sequencing data

| Audience | Computational Skills | Prerequisites | Duration |
:----------|:----------|:----------|:----------|
| Biologists | Beginner bash | Introduction to the command-line interface | 3 hour workshop (~3 hours of trainer-led time)|


### Description

This repository has teaching materials for a **3 hour**, hands-on **Accessing genomic reference and experimental sequencing data** workshop led at a relaxed pace. 

For many types of sequencing analyses, we need access to public data stored in various databases and repositories. This workshop will discuss types of genomic reference data available through public databases such as Ensembl, NCBI, and UCSC, and step through how to find and download this data. The workshop will also explore how to find and download publicly available experimental data, such as data (FASTQ files and count matrices) from published papers, using GEO and the SRA repositories. While most of the workshop will access data using a web browser, downloading data from the SRA will require beginner knowledge of the command-line interface. 

### Learning Objectives

* Understanding what is a genome build
* Identifying differences in reference data available from Ensembl, NCBI, and UCSC
* Finding and downloading experiment-appropriate genome reference data
* Finding and downloading publically available experimental sequence data

> These materials are developed for a trainer-led workshop, but also amenable to self-guided learning.


### Contents

| Lessons            | Estimated Duration |
|:------------------------|:----------|
|[Accessing publically available NGS experimental data from Gene Expression Omnibus (GEO)](lessons/accessing_public_experimental_data.md)| 45 min |
|[Accessing publically available NGS experimental data from the Sequence Read Archive (SRA)](lessons/downloading_from_SRA.md)| 60 min |
|[Accessing genome reference data](lessons/accessing_genome_reference_data.md) | 60 min |

### Installation Requirements

***Mac users:***
No installation requirements.

***Windows users:***
[GitBash](https://git-scm.com/download/win)


